### ISSUE TITLE
Feature Request (HLM, PCR, Ridge etc...)

### ISSUE THREAD

**Issue by Farscape1** [jasp-stats/jasp-desktop#2733]
Hi,

I am wondering if JASP is planning to add multinomial regression, multi-level modelling, and things like principal component regression and lasso.

I know it is a lot of feature request but the use of multiomial, multilevel, and PCR is becoming frequent in our field (medical).

Thanks,

**Comment by EJWagenmakers** [comment]
Definitely. When exactly we'll get to this is not 100% clear, as we also have other fish to fry, but some of this is relatively straightforward and some other stuff is already on our long-term to-do list.
 
Cheers,
E.J.
